### ISSUE TITLE
Add victory conditions for skirmish UI ticket

### ISSUE THREAD

**Issue by GoogleFrog** [ZeroK-RTS/Zero-K#2467]
The victory condition is currently to kill everything. I think we also have a commends toggle. This ticket wants more victory conditions: https://github.com/ZeroK-RTS/Chobby/issues/299

They both sound like a good idea. Victory type would then a modoption with four options: Assassinate, Factory Assassinate, Overwhelming Annihilation, Complete Annihilation.

**Comment by CrazyEddieTK** [comment]
I suggest calling them Assassination, Domination, Annihilation, and Complete Annihilation.

The chobby ticket wants them in the Skirmish UI where they can easily be selected. For the basic UI, just default to Domination and don't offer a choice. For the advanced UI, have a radio button to select them without forcing the player to mess around with modoptions.

**Comment by GoogleFrog** [comment]
Yes, that may be what chobby wants. You should say this on the chobby ticket so it is not forgotten. The gameside implementation will be a modoption with four options, when that is done I will consider this ticket complete. Special defaults and UI for modoptions are left up to chobby.

**Comment by GoogleFrog** [comment]
The non-modoption version exists https://github.com/ZeroK-RTS/Zero-K/commit/95928fb3371141dffd923adcbf30a41d6ccee88a
